README.md

```diff
@@ -151,18 +154,22 @@ to match a non-virtual device with the profile.
 
 ### `interface_name`
 
-For type `ethernet`, this option restricts the profile to the
-given interface by name. This argument is optional and by default
-a profile is not restricted to any interface by name.
-Note that with [persistent interface naming](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Networking_Guide/ch-Consistent_Network_Device_Naming.html),
+For type `ethernet`, this option restricts the profile to the given interface
```

Comment:
I guess it is not only for `ethernet` but for any physical interface type (e.g. `infiniband`)?

Reply:
Good catch, infiniband is the other network device type that supports this. AFAICS, there is no other physical interface type supported in the code.

Reply:
What about
> For physical interface types (`ethernet` and `infiniband`)

to make clear what is the logic of supporting it for some types and not for others.

Reply:
The `ethernet` also works for virtual interfaces that created outside the role, e.g. veth interfaces and probably also bridges or maybe dummy interfaces. Then the device will not be created by the role but only be configured like a regular ethernet interfaces as far as I can tell.